tests/integration/distribution.go

```diff
@@ -934,7 +934,7 @@ func (s *CCVTestSuite) prepareRewardDist() {
 	s.coordinator.CommitNBlocks(s.consumerChain, uint64(blocksToGo))
 }
 
-func (s *CCVTestSuite) TestAllocateTokensToValidator() {
+func (s *CCVTestSuite) TestAllocateTokensToConsumerValidators() {
```

Comment:
This test was practically testing `AllocateTokensToConsumerValidators` so renamed it.